### ISSUE TITLE
Centos Docker: Operation not permitted (bundled/zeromq/src/thread.cpp:269)

### ISSUE THREAD

**Issue by mfdhsoc** [log2timeline/plaso#4184]
**Description of problem:**

It looks that the issue above observed in the previous Docker container still persists.

**Command line and arguments:**

Just tried to test modifying the example on the Plaso web site:
docker run -v /evidence/:/evidence log2timeline/plaso log2timeline --storage-file /evidence/mylaptop.plaso /evidence/mylaptop

**Source data:**

Windows 11 triage data collected using CyLR.

**Plaso version:**

plaso - log2timeline version 20220724

**Operating system Plaso is running on:**

Rocky Linux 8.6

**Installation method:**

docker pull log2timeline/plaso

**Debug output/tracebacks:**

2022-08-09 07:26:35,978 [INFO] (MainProcess) PID:7 <data_location> Determined data location: /usr/share/plaso
2022-08-09 07:26:35,989 [INFO] (MainProcess) PID:7 <artifact_definitions> Determined artifact definitions path: /usr/share/artifacts
Operation not permitted (bundled/zeromq/src/thread.cpp:269)
/usr/local/bin/plaso-switch.sh: line 17:     7 Aborted                 (core dumped) /usr/bin/log2timeline.py --unattended "${@:2}"

**Comment by joachimmetz** [comment]
This looks like to be an issue related to libzmq/pyzmq specific to Windows.

**Comment by joachimmetz** [comment]
> It looks that the issue above observed in the previous Docker container still persists.

This looks like the first report of this issue. We cannot address issues that are not reported.

**Comment by mfdhsoc** [comment]
@joachimmetz 
> This looks like the first report of this issue. We cannot address issues that are not reported.
It might have been my misunderstanding but I saw the similar issue has been fixed in the version of 20220724, in a readthedocs web page. Apologies if my misunderstanding made you confused.

You chenged the title but I didn't use the Windows Docker but used CentOS Docker.

**Comment by joachimmetz** [comment]
> You chenged the title but I didn't use the Windows Docker but used CentOS Docker.

@mfdhsoc can you make sure to provide ALL RELEVANT details in your issue description

**Comment by mfdhsoc** [comment]
I just started to use the Plaso Docker container and don't have more information. I tried --debug but it didn't show much information. Since the CyLR result (unzipped) may contain our customer names, it cannot be provided. As you may know, it consists of $MFT, registry hives, event logs, and so on.  I put --troubles result just in case, while it looks quite OK.

#> docker run log2timeline/plaso log2timeline --troubles
2022-08-10 02:50:13,028 [INFO] (MainProcess) PID:7 <data_location> Determined data location: /usr/share/plaso
Using Python version 3.10.4 (main, Jun 29 2022, 12:14:53) [GCC 11.2.0]

Path: /usr/bin/log2timeline.py

plaso - log2timeline version 20220724

Checking availability and versions of dependencies.
[OK]            artifacts version: 20220615
[OK]            bencode
[OK]            certifi version: 2020.06.20
[OK]            cryptography version: 3.4.8
[OK]            dateutil version: 2.8.2
[OK]            defusedxml version: 0.7.1
[OK]            dfdatetime version: 20220131
[OK]            dfvfs version: 20220430
[OK]            dfwinreg version: 20211207
[OK]            dtfabric version: 20220219
[OK]            future version: 0.18.2
[OK]            lz4 version: 4.0.2
[OK]            opensearchpy
[OK]            pefile version: 2022.5.30
[OK]            psutil version: 5.9.1
[OK]            pybde version: 20220121
[OK]            pycreg version: 20210625
[OK]            pyesedb version: 20210424
[OK]            pyevt version: 20210424
[OK]            pyevtx version: 20210525
[OK]            pyewf version: 20140814
[OK]            pyfsapfs version: 20220709
[OK]            pyfsext version: 20220529
[OK]            pyfshfs version: 20220709
[OK]            pyfsntfs version: 20220220
[OK]            pyfsxfs version: 20220706
[OK]            pyfvde version: 20220125
[OK]            pyfwnt version: 20210906
[OK]            pyfwsi version: 20220123
[OK]            pylnk version: 20210417
[OK]            pyluksde version: 20220121
[OK]            pymodi version: 20210807
[OK]            pymsiecf version: 20210420
[OK]            pyolecf version: 20210512
[OK]            pyparsing version: 2.4.7
[OK]            pyphdi version: 20220301
[OK]            pyqcow version: 20210419
[OK]            pyregf version: 20220131
[OK]            pyscca version: 20210419
[OK]            pysigscan version: 20220124
[OK]            pysmdev version: 20210418
[OK]            pysmraw version: 20210807
[OK]            pytsk3 version: 20211111
[OK]            pytz
[OK]            pyvhdi version: 20210425
[OK]            pyvmdk version: 20210807
[OK]            pyvsgpt version: 20211115
[OK]            pyvshadow version: 20211114
[OK]            pyvslvm version: 20210807
[OK]            redis version: 3.5.3
[OK]            requests version: 2.25.1
[OK]            six version: 1.16.0
[OK]            urllib3 version: 1.26.5
[OK]            xattr version: 0.9.9
[OK]            xlsxwriter version: 3.0.3
[OK]            yaml version: 5.4.1
[OK]            yara version: 4.2.0
[OK]            zmq version: 23.2.0

Also see: https://plaso.readthedocs.io/en/latest/sources/user/Troubleshooting.html

**Comment by joachimmetz** [comment]
@mfdhsoc which version of Docker are you running?

**Comment by mfdhsoc** [comment]
Docker version is 20.10.9.

**Comment by joachimmetz** [comment]
Can you try a more recent version of Docker? Per https://stackoverflow.com/questions/70087344/python-in-docker-runtimeerror-cant-start-new-thread

```
This is because the default seccomp profile of Docker 20.10.9 is not adjusted to support the clone() syscall wrapper of glibc 2.34 adopted in Ubuntu 21.10 and Fedora 35.
```

Which sounds like a plausible cause, so let's see if you can update Docker first

**Comment by mfdhsoc** [comment]
It looks that the issue was resolved by updating to Docker 20.10.10. Core Dump didn't happen and my process went into a heavy-weight process of Plaso.

Thank you for your support @joachimmetz !
